modules/cnf-programming-the-sriov-operator.adoc

```diff
@@ -0,0 +1,398 @@
+// CNF-1498 Validate and Document Intel SRO and SRIOV FEC Operator
+// Module included in the following assemblies:
+//
+// *cnf-optimize-data-performance-n3000.adoc
+
+[id="configuring-the-sr-iov-fec-operator-intel-fpga-pac-n3000_{context}"]
```

Comment:
Please match the ID to the file name.

https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#module-file-metadata